packages/dnb-eufemia/src/components/date-picker/__tests__/DatePicker.test.tsx

```diff
@@ -2650,6 +2650,49 @@ describe('Custom text for buttons', () => {
   })
 })
 
+describe('DatePickerPortal', () => {
```

Comment:
```suggestion
describe('DatePickerPortal', () => {
```

A `skipPortal` test would be good to include.